purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/events/CustomerCenterEvent.kt

```diff
@@ -0,0 +1,45 @@
+package com.revenuecat.purchases.customercenter.events
+
+import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI
+import com.revenuecat.purchases.utils.serializers.DateSerializer
+import com.revenuecat.purchases.utils.serializers.UUIDSerializer
+import kotlinx.serialization.Serializable
+import java.util.Date
+import java.util.UUID
+
+/**
+ * Type representing a customer center event and associated data. Meant for RevenueCatUI use.
+ */
+@ExperimentalPreviewRevenueCatPurchasesAPI
+@Serializable
+data class CustomerCenterEvent(
```

Comment:
This is good but we've recently decided to start moving away from data classes since they introduced methods that may introduce breaking changes down the line if it changes. Instead, we've been using the `Poko` library to autogenerate some of the methods that might be useful. In this case, the changes are propose are:
```suggestion
@Poko
@Serializable 
class CustomerCenterEvent(
```
It would be the same for the other inner classes.